Sources/Logto/Types/IdTokenClaims.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  IdTokenClaims.swift
+//  
+//
+//  Created by Gao Sun on 2022/1/7.
+//
+
+import Foundation
+
+public struct IDTokenClaims {
```

Comment:
`IdTokenClaims`: `d` should be in lower case according to the [documentation](https://www.notion.so/silverhand/9c1fbcd630fc44adaa45a6d7d686326a?v=3cea92e2906048d287a8648b76c73c65)

Reply:
good call